control-plane/csi-driver/src/bin/node/filesystem_ops.rs

```diff
@@ -132,10 +132,14 @@ pub(crate) trait FileSystemOps: Send + Sync {
 
 #[async_trait]
 impl FileSystemOps for Ext4Fs {
-    async fn create(&self, device: &str) -> Result<(), Error> {
+    async fn create(&self, device: &str, format_options: &str) -> Result<(), Error> {
         let binary = "mkfs.ext4";
-        let output = Command::new(binary)
-            .arg(device)
+        let mut cmd = Command::new(binary);
+        cmd.arg(device);
+        if !format_options.is_empty() {
+            cmd.arg(format_options);
```

Comment:
do we not need to split into a vector like for xfs?

Reply:
Are you asking should we split here also?

During xfs arg testing noticed that `"mkfs.xfs" "/dev/nvme0n1" "-m bigtime=0 -m inobtcount=0"` resulted in error. 

After splitting into vec it worked.

`"mkfs.xfs" "-m" "bigtime=0" "-m" "inobtcount=0" "/dev/nvme0n1"`

Ive done that for Ext4 and BtrFs also.